policyengine_us/variables/gov/states/ct/tax/income/subtractions/ct_magi_excess_over_base.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class ct_magi_excess_over_base(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Excess of modified adjusted gross income and social security over base amount calculation for the Connecticut social security subtraction"
+    reference = (
+        # Connecticut General Statutes, Chapter 229, Sec. 12-701, (20), (b), (x), (iii) and (iv)
+        "https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-701"
+        # Section 86(b)(1) of the Internal Revenue Code
+        "https://www.law.cornell.edu/uscode/text/26/86"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    # Code extracted from tax_unit_taxable_social_security
+    def formula(tax_unit, period, parameters):
+        ss = parameters(period).gov.irs.social_security.taxability
```

Comment:
```suggestion
        p = parameters(period).gov.irs.social_security.taxability
```
Or p_ss

By convention we use p for parameter subtrees